CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
   - Already registered connectors will be updated automatically, this process can take up to 24 hours
 - Added a message when the CaaS request feature is not available
 - Catalog: Removed dataspace filter when only one dataspace is known
+- Catalog: Merged Organization ID and name filter into one
```

Comment:
`- Catalog: Connector Filter now also displays names and organization names`

Reply:
That doesn't really describe the change though. The names were there before, they were just separate filters